src/main/kotlin/com/the_chance/data/tables/ProductTable.kt

```diff
@@ -1,8 +1,10 @@
 package com.the_chance.data.tables
 
-import org.jetbrains.exposed.dao.id.UUIDTable
+import org.jetbrains.exposed.dao.id.LongIdTable
 
-object ProductTable : UUIDTable() {
+object ProductTable : LongIdTable() {
     val name = text("name")
     val price = double("price")
+    val quantity = text("quantity")
+    val isDeleted = bool("isDeleted")
```

Comment:
Yes, Here in this line you can make it take default value as false So, You don't need to initiate it when you create product.
like that ===> 
val isDeleted = bool("isDeleted").default(false)